sql/core/src/main/scala/org/apache/spark/sql/execution/command/AlterTableAlterViewCommandParser.scala

```diff
@@ -29,22 +29,26 @@ import org.apache.spark.sql.types.StructType
 
 
 /**
- * Helper object to parse alter table commands.
+ * Helper object to parse alter table or alter view commands.
  */
-object AlterTableCommandParser {
+object AlterTableAlterViewCommandParser {
```

Comment:
I see. Thank you for letting me know it. 

Will first do the change. Then, update it if necessary.